src/sema/expression.rs

```diff
@@ -6326,118 +6323,119 @@ fn resolve_using(
     ns: &mut Namespace,
     resolve_to: ResolveTo,
 ) -> Result<Option<Expression>, ()> {
-    // first collect all possible libraries that match the using directive type
+    // first collect all possible functions that could be used for using
     // Use HashSet for deduplication.
```

Comment:
`expression.rs` has more than 7,000 lines. If you can, please place the `fn resolve_using` in `using.rs`